### PR TITLE
Convert Mysterion to Flow

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

File: src/app/mysterion.js [hunk 1]
```diff
@@ -15,35 +15,78 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {app} from 'electron'
+// @flow
+
+import { app, BrowserWindow } from 'electron'
 import trayFactory from '../main/tray/factory'
-import {logLevels as processLogLevels} from '../libraries/mysterium-client'
+import { Installer, logLevels as processLogLevels } from '../libraries/mysterium-client'
```
Comment (shroomist) [inline src/app/mysterion.js]: spaces in curlies - I find that  @ignasbernotas has different config for this.
I like it this way better, it also makes navigation a bit easier.

File: src/app/mysterion.js [hunk 2]
```diff
@@ -146,7 +190,7 @@ class Mysterion {
     }
   }
 
-  _getSendFunction (browserWindow) {
+  _getSendFunction (browserWindow: Object) {
```
Comment (shroomist) [inline src/app/mysterion.js]: (browserWindow: BrowserWindow)

File: src/app/mysterion.js [hunk 3]
```diff
@@ -291,7 +336,7 @@ class Mysterion {
     this.monitoring.start()
   }
 
-  _onProcessReady (callback) {
+  _onProcessReady (callback: Function) {
```
Comment (shroomist) [inline src/app/mysterion.js]: let's consolidate this to (callback: () => void)

File: src/app/mysterion.js (hunk 1 above)
Comment (ignasbernotas) [reply src/app/mysterion.js]: 👍 fixed it on my end, thanks for noticing:)

File: src/app/mysterion.js (hunk 2 above)
Comment (donce) [reply src/app/mysterion.js]: Fixed.

File: src/app/mysterion.js (hunk 3 above)
Comment (donce) [reply src/app/mysterion.js]: Fixed.

**Pull request merged**